### ISSUE TITLE
Texture du fond de carte pixelisée sur Mac

### ISSUE THREAD

**Issue by meschultheiss** [EPFL-ENAC/LASUR-swiss-proximity#59]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Le fond de carte est pixelisé et n'apparait pas de très bonne qualité sur mon MacBook. Quelque chose à voir avec l'écran retina ?

**Comment by guilbep** [comment]
Yes @pierreleripoll  pour info voir https://github.com/EPFL-ENAC/alice-perl-eerl-urbtrees/pull/18 et https://github.com/EPFL-ENAC/alice-perl-eerl-urbtrees/issues/16
